docs/modules/ROOT/pages/vshn-managed/keycloak/usage.adoc

```diff
@@ -0,0 +1,62 @@
+= Using a Keycloak service with FQDN
+
+== Issue a Keycloak instance
+The YAML code below creates the service `VSHNKeycloak` with a Full Qualified Domain Name(FQDN).
+
+[source,yaml]
+----
+apiVersion: vshn.appcat.vshn.io/v1
+kind: VSHNKeycloak
+metadata:
+  name: keycloak-app1-prod
+  namespace: prod-app
+spec:
+  parameters:
+    service:
+      version: "23"
+      fqdn: my-keycloak.example.com # <1>
+    size:
+      plan: standard-2
+  writeConnectionSecretToRef:
+    name: keycloak-creds-connection # <2>
+----
+
+<1> Your full qualified domain name
+<2> Credentials to access the keycloak
+
+== Configure your DNS server
+
+On APPUiO Cloud we provide you with a cert-manager setup which you can use to create, sign, install and renew certificates for your domains running on APPUiO Cloud.
+
+To create a certificate for the Keycloak FQDN in your domain, you need to create a CNAME record in your domain’s DNS pointing to your APPUiO Zone’s well-defined cname record.
+
+[source, text]
+----
+my-keycloak  IN  CNAME  cname.cloudscale-lpg-0.appuio.cloud.
+----
+
+== Access Keycloak
```

Comment:
Maybe mention here somewhere that they can change the password for the admin account if they want. But also mention that the credential secret won't be valid then anymore.

Also make a warning or note thingy: they should not change the password of the `internaladmin` user! It's used internally for various scripts and configurations. They risk breaking their instance if they do that.